src/authorization.rs

```diff
@@ -450,7 +450,6 @@ impl Verifiable for Invocation {
 
             // check capabilities are supported by parents
             if !res.iter().any(|c| *c) {
-                // TODO should not return 500
```

Comment:
is this not actually the case?

Reply:
ah, it was causing failures in the sdk tests which expected 403 and I saw these todos so fixed it up. the error type for that code path was very vague but does distinguish only between unauthorised and other/"actual error", I figured it would be better to externalise that distinction for the clients benefit